pubspec.lock

```diff
@@ -84,42 +84,42 @@ packages:
       name: firebase_auth
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "3.3.16"
+    version: "3.3.17"
```

Comment:
As I mentioned in the teams channel, I have no idea why these lock file changes are here but I figured I would commit them _just in case_